project/profiles/capacity4more/modules/c4m/demo/c4m_demo_content/csv/node/project.csv

```diff
@@ -1,3 +1,3 @@
\ No newline at end of file
+ID,UID,Title,PURL,Image,Banner,Duration,Budget,Currency,Funding,Partner,CRIS Decision,CRIS Contract,Body,Topics,Geo,Groups,Projects,Membership_Request,Access
+project_1,user_3,Lusail City,lusail-city,lusail.jpg,lusail-banner.jpg,2012-07-01 00:00:00;2022-05-31 00:00:00,45000000000,USD,,,CRIS-2012-001,CRIS 000201207010001,"One of the biggest developments in the Middle East. The new Lusail City to the northern side of Qatar will cover a 38km2 site which will eventually house up to 200,000 people. The $45bn project is being developed in several different zones containing residential, commercial and financial districts. It will include a new waterfront, a $275m mall, 22 hotels, two golf courses and four offshore islands. The city will also be home to the Lusail Iconic Stadium - the 80,000-seat showpiece stadium for Qatar's 2022 World Cup and the venue for the tournament's final. The project, which is due to complete in 2020, is being overseen by Qatari Diar - the real estate investment arm of Qatar?s sovereign wealth fund.",t_earth,Qatar,group_1|group_6,,0,0
+project_2,user_1,Human Genome Project,human-genome,dna.png,dna-banner.jpg,1990-01-01 00:00:00;2003-12-05 00:00:00,16000000,EUR,Qatari Diar Real Estate Investment Company,,CRIS-1990-001,CRIS 000199001010003,"The Human Genome Project (HGP) is an international scientific research project with the goal of determining the sequence of chemical base pairs which make up human DNA, and of identifying and mapping all of the genes of the human genome from both a physical and functional standpoint. It remains the world's largest collaborative biological project. The project was proposed and funded by the US government ; planning started in 1984, the project got underway in 1990, and was declared complete in 2003. A parallel project was conducted outside of government by the Celera Corporation, or Celera Genomics, which was formally launched in 1998. Most of the government-sponsored sequencing was performed in universities and research centres in China, France, Germany, Japan, Spain, the United Kingdom, and the United States",t_wind,Worldwide,group_5,project_1,1,0
```

Comment:
@anvmn 
I think you should remove one of the `,` after the `Qatari Diar Real Estate Investment Company`